UpdateExpiredCgn/function.json

```diff
@@ -1,7 +1,7 @@
 {
   "bindings": [
     {
-        "schedule": "0 0 * * * *",
+        "schedule": "0 0 0 * * *",
```

Comment:
Why this change?

Reply:
Ok, is executed one time every day. We are considering the [timezone](https://docs.microsoft.com/it-it/azure/azure-functions/functions-bindings-timer?tabs=csharp#ncrontab-time-zones)?

Reply:
Basically the nCrontab is expressed as UTC so maybe we have to shift the schedule by 1 hour :)

Reply:
Maybe is not enough, we should consider solar/legal time or, if we haven't a _Linux Consumption plan_ we can set `WEBSITE_TIME_ZONE` into the function configuration.

Reply:
We will provide timezone with `WEBSITE_TIME_ZONE` thanks to @uolter